package.json

```
@@ -0,0 +1,77 @@
+{
+  "name": "aedes-server-factory",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.1.0",
```

Until we don't have an `aedes@1.0.0` I prefer to keep this `0.x.x` (maybe after this we could think about an aedes 1.0.0)

---

Ok, i just wanted to keep the same version as protocol-decoder